### PR TITLE
86956b Additional copy updates - form 10-7959a

### PR DESCRIPTION
**Pull request by michaelclement**
## Are you removing, renaming or moving a folder in this PR?
- [X] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

Updates copy on a few pages and fixes a `depends` function to make sure we show the right upload screens.

- Update pharmacy claim page wording
- Update EOB upload page wording
- Update URL for insurance intro list loop page
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86956

## Testing done

- Manual
- Verified unit tests run and pass

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Insurance Intro page URL  |        |url|
| Pharmacy claim upload |prescrip_claim_before|prescrip_claim_after|
|Medical claim upload|Screenshot 2024-07-15 at 12 26 02|med_claim_page_after|
| EOB upload | |eob_upload_after|

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-7959a/chapters/claimInformation.js [hunk 1]
```diff
@@ -197,8 +195,8 @@ export const eobUploadSchema = isPrimary => {
           }`;
         },
         ({ formData }) => {
-          const name = nameWording(formData, true, true, true);
-          const yourOrTheir = name === 'your' ? name : 'their';
+          const name = nameWording(formData, true, false, true);
+          const yourOrTheir = name.toLowerCase() === 'your' ? name : 'their';
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959a/chapters/claimInformation.js:199]: adjusted `nameWording` params so it returns a lowercased "your", and updated `yourOrTheir` check to be case-insensitive as an extra precaution.

File: src/applications/ivc-champva/10-7959a/config/form.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ const formConfig = {
         page7: {
           path: 'medical-claim-upload',
           title: 'Supporting documents',
-          depends: formData => get('claimIsWorkRelated', formData),
+          depends: formData => get('claimType', formData) === 'medical',
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959a/config/form.js:216]: Fixed this so it checks `claimType` (what we wanted all along)

File: src/applications/ivc-champva/10-7959a/config/form.js [hunk 3]
```diff
@@ -237,7 +237,8 @@ const formConfig = {
           depends: formData =>
             get('hasOhi', formData) &&
             get('claimType', formData) === 'medical' &&
-            get('policies', formData).length > 1,
+            get('policies', formData) &&
+            formData?.policies?.length > 1,
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959a/config/form.js:241]: Checks if the array exists, and then uses conditional chaining to check if the length is over one. Previously, if the user hadn't actually added any policies we could get a console warning. This prevents that and makes the whole check more explicit.

**Review by balexandr — APPROVED** [review]

**Review by cloudmagic80 — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**